vcpkg.json

```
@@ -0,0 +1,14 @@
+{
+    "name": "voxelengine",
+    "version-string": "1.0.0",
+    "dependencies": [
+      "opengl",
+      "openal-soft",
+      "glfw3",
+      "glew",
+      "glm",
+      "libspng",
+      "zlib",
+      "luajit"
+    ]
+  }
```

```suggestion
  }

```
Don't forget to add the end of line (EOF) character (i.e. "enter" key) at the end of the file.